wgsl/index.bs

```diff
@@ -2036,16 +2036,27 @@ Some consequences of the restrictions structure member and array element types a
     | [=syntax/attribute=] * [=syntax/member_ident=] [=syntax/colon=] [=syntax/type_decl=]
 </div>
 
-WGSL defines the following attributes that can be applied to structure members:
+The following attributes can be applied to structure members:
  * [=attribute/align=]
  * [=attribute/builtin=]
  * [=attribute/location=]
  * [=attribute/interpolate=]
  * [=attribute/invariant=]
  * [=attribute/size=]
 
-Note: Layout attributes may be required if the structure type is used
-to define a [=uniform buffer=] or a [=storage buffer=]. See [[#memory-layouts]].
+Attributes [=attribute/builtin=] [=attribute/location=] [=attribute/interpolate=], and [=attribute/invariant=]
+are [=IO attributes=].
+An [=IO attribute=] on a member of a structure |S| has effect only when
```

Comment:
```suggestion
An [=IO attribute=] on a member of a structure |S| has an effect only when
```

Reply:
Separately, I think |S| (and later |T|) are the wrong markdown to use here since this is not an algorithm. Suggest \_S\_  (_S_) if italics are desired still or \`S\` (`S`) for code.